ambry-clustermap/src/main/java/com.github.ambry.clustermap/Replica.java

```diff
@@ -32,6 +32,7 @@
 class Replica implements ReplicaId {
   private final Partition partition;
   private Disk disk;
+  private volatile boolean isStopped = false;
```

Comment:
are you sure this variable needs to be `volatile`?  I don't see any ordering issues.

Reply:
Not quite sure but it's safe to make it volatile to avoid issues on visibility of variable in current memory model.